dotcom-rendering/src/components/Discussion/CommentContainer.test.tsx

```diff
@@ -74,6 +74,8 @@ describe('CommentContainer', () => {
 				setError={() => {}}
 				userNameMissing={false}
 				setUserNameMissing={() => {}}
+				previewBody=""
+				setPreviewBody={() => undefined}
```

Comment:
Small consistency note, other props use the `() => {}` pattern

Reply:
copied this from somewhere else where the pattern is `()=>undefined` 🙅 . I'll keep it consistent in this file!